leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -20,16 +20,17 @@
 public final class AnalysisResult implements Serializable {
 
   public static AnalysisResult noLeak(long analysisDurationMs) {
-    return new AnalysisResult(false, false, null, null, null, analysisDurationMs);
+    return new AnalysisResult(false, false, null, null, null, 0, analysisDurationMs);
```

Comment:
@pepyakin I'm strictly against any non-JavaDocs comments in code. They're not consistent, they're not enforced, they're utilize features of IDE. So, either `Builder` or at least `JavaDoc` for constructor is better.